src/components/inspectors/TimerExpression.vue

```diff
@@ -168,15 +168,18 @@ export default {
       return 'You must select at least one day.';
     },
     iso8606Expression() {
```

Comment:
I think this needs a better name. How about `iso8601FormattedDate`? It's not clear what an 'Expression' is in this context and there's a typo in the name ISO 8606 deals with plastics (https://standards.globalspec.com/std/694731/ISO%208606)...